### DISCUSSION TITLE
Can I use + and - buttons instead of a slider to control temperature in HA?

### DISCUSSION THREAD

**Discussion by GammaScorpii** [omerfaruk-aran/esphome_samsung_hvac_bus#124] [General]
The slider is very fiddly and cumbersome to use, considering at most I would change the temperature by 2 degrees. Last night I woke up to the AC blasting set at 29c, because my finger must have brushed past it and set it to almost max temp.

**Comment by lanwin** [comment]
Hi @GammaScorpii do you use the climate component or separate sensors? At least the climate component have a + and - button.

**Reply by GammaScorpii** [reply]
I'm not sure actually!

This is all I can see other than visiting samsung_ac.local:

![Screenshot 2024-05-06 175037](https://github.com/lanwin/esphome_samsung_ac/assets/28531783/f3729e08-fa02-46a0-a692-f50387086b16)

**Reply by lanwin** [reply]
Please check https://github.com/lanwin/esphome_samsung_ac/blob/ead34499ef973496745d83c04e72a4c6e6227a7a/example.yaml#L68

**Reply by GammaScorpii** [reply]
Perfect, thank you. For some reason I had commented that part out during testing and never go around to trying it out.
